platforms/quarkus-bom.yaml

```diff
@@ -30,3 +30,5 @@ exclude-versions:
 pinned-streams:
 - "2.13"
 - "3.2"
```

Comment:
LTS does not necessary make it pinned, right?

Reply:
Correct, they are two separate concepts